clusterloader2/pkg/measurement/util/workerqueue/workerqueue.go

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
This has merged already, can you please rebase?

Reply:
Done.